src/Scaffold/PresetPackage.php

```diff
@@ -59,6 +60,8 @@ public function runInstaller($console)
             throw $e;
         } catch (Exception $e) {
             throw new Exception("The 'init.php' file for this preset contains errors.");
+        } catch (Error $e) {
+            throw new Exception("The 'init.php' file for this preset contains errors.");
```

Comment:
Is this specific to PHP 8?

Reply:
I'm not 100% certain, but in PHP 8 it was throwing an `Error` specific to improper php formatting and not the expecting `Exception` as it was before.